brickstrap.sh

```diff
@@ -428,15 +428,71 @@ function brp_preseed_debconf() {
     rm "$(br_rootfs_dir)/tmp/debconfseed.txt"
 }
 
+#
+# Ensure that a 'usable' /etc/shells file exists for the add-shell utility.
```

Comment:
Yes, specifically debianutils 4.6.

It's because the maintainer of `debianutils` is looking for ways around using `awk` and his first attempt using `grep '^'` didn't work properly. I've also filed a bug about this here: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=812784

In general I think it's a good idea for brickstrap to do this because `dash` is such a critical package. The alternative of potentially hiding failure to configure dash correctly is worse than having to carry a straightforward enough workaround.